docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_ad_auth_arm_template/_index.md

```diff
@@ -0,0 +1,224 @@
+
+  ![Screenshot showing the PowerShell logon script run](./06.png)
+
+  ![Screenshot showing the PowerShell logon script run](./07.png)
+
+  ![Screenshot showing the PowerShell logon script run](./08.png)
+
+  ![Screenshot showing the PowerShell logon script run](./09.png)
+
+  ![Screenshot showing the PowerShell logon script run](./10.png)
+
+  ![Screenshot showing the PowerShell logon script run](./11.png)
+
+  ![Screenshot showing the PowerShell logon script run](./12.png)
+
+  ![Screenshot showing the PowerShell logon script run](./13.png)
+
+  ![Screenshot showing the PowerShell logon script run](./14.png)
+
+  ![Screenshot showing the PowerShell logon script run](./15.png)
+
+  ![Screenshot showing the PowerShell logon script run](./16.png)
+
+  ![Screenshot showing the PowerShell logon script run](./17.png)
+
+    ![Screenshot showing the post-run desktop](./18.png)
+
+- Since this scenario is deploying the Azure Arc Data Controller and SQL Managed Instance, you will also notice additional newly deployed Azure resources in the resources group (at this point you should have **16 various Azure resources deployed**. The important ones to notice are:
+
+  - _Azure Arc-enabled Kubernetes cluster_ - Azure Arc-enabled data services deployed in directly connected are using this type of resource in order to deploy the data services [cluster extension](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-extensions) as well as for using Azure Arc [Custom locations](https://docs.microsoft.com/azure/azure-arc/kubernetes/conceptual-custom-locations).
+
+  - _Custom location_ - provides a way for tenant administrators to use their Azure Arc-enabled Kubernetes clusters as target locations for deploying Azure services instances.
+
+  - _Azure Arc Data Controller_ - The data controller that is now deployed on the Kubernetes cluster.
+
+  > **NOTE: Azure resource will not be created for SQL Managed Instance with AD authentication created in the scenario using YAML. Azure resources are created only when using ARM template or deployed SQL Managed Instance using Azure Portal**.
+
+    ![Screenshot showing additional Azure resources in the resource group](./19.png)
+
+- As part of the automation, SQL Managed Instance endpoints desktop shortcut is created to view connection information and login to the SQL Managed Instance using Windows authentication. Copy the endpoint information to login to the SQL server.
+
+  ![Screenshot showing SQL Managed Instance endpoints and database server credentials](./20.png)
+
+- As part of the automation, SQL Server Management Studio and Azure Data Studio is installed along with the _Azure Data CLI_, _Azure CLI_, _Azure Arc_ and the _PostgreSQL_ extensions. Open SQL Server Management Studio to connect to SQL Managed Instance using Windows Authentication.
```

Comment:
Data studio settings file is not configured for this instance. You would need to edit the settingsTemplate.json to allow for it to be configured to connect to the AD-joined instance

Reply:
This needs additional changes to include sections on-demand based on what user chooses. Right now, I see PostgreSQL connection even though I did not deploy it. I will make changes to included only what deploys.